pep/app/pods/browse/journal/volume/route.ts

```diff
@@ -0,0 +1,13 @@
+import Route from '@ember/routing/route';
+
+export default class BrowseJournalVolume extends Route {
+    model(params: { volume_number: string }) {
+        const journalParams = this.paramsFor('browse.journal') as { pep_code: string };
```

Comment:
just cleanup, but in other projects we usually type the params as an exported interface on the route file that it belongs to e.g.
```ts
// in browse/journal/route.ts
export interface BrowseJouralParams {
    pep_code: string
}
```